libs/datamodel/core/src/lib.rs

```diff
@@ -201,9 +201,9 @@ fn load_sources(
 //
 
 /// Renders to a return string.
-pub fn render_datamodel_to_string(datamodel: &dml::Datamodel) -> String {
+pub fn render_datamodel_to_string(datamodel: &dml::Datamodel, configuration: Option<&Configuration>) -> String {
     let mut writable_string = String::with_capacity(datamodel.models.len() * 20);
-    render_datamodel_to(&mut writable_string, datamodel, None);
+    render_datamodel_to(&mut writable_string, datamodel, configuration);
```

Comment:
Is this what the IE was using? I'm finding the rendering codepath hard to picture.

Reply:
Yeah, this is kind of confusing... So we have the function that renders config and dml that's using the config specialties for rendering and then we have the one that renders only the dml that didn't get the config at all. Now we kind of need to know the connector and preview features in this point too...

Reply:
Ok yes now with a bit more time and context I think I understand.